spock-specs/src/test/groovy/org/spockframework/docs/extension/IgnoreIfDocSpec.groovy

```diff
@@ -0,0 +1,40 @@
+package org.spockframework.docs.extension
+
+import org.spockframework.runtime.extension.builtin.PreconditionContext
+import spock.lang.IgnoreIf
+import spock.lang.Specification
+
+class IgnoreIfDocSpec extends Specification {
+// tag::example-a[]
+  @IgnoreIf({ System.getProperty("os.name").toLowerCase().contains("windows") })
```

Comment:
we have the `sys` shortcut

Reply:
We even have an `os` shortcut, but it shouldn't be used here.
It's the first example that shows that you can formulate a condition.
The second example shows that you can write it more concise using our helpers.
I actually just copied over the existing examples.